activemodel/lib/active_model/naming.rb

```diff
@@ -8,11 +8,12 @@ module ActiveModel
   class Name
     include Comparable
 
-    attr_accessor :singular, :plural, :element, :collection,
-      :singular_route_key, :route_key, :param_key, :i18n_key,
-      :name
+    attr_accessor :singular, :plural, :uncountable,
```

Comment:
I don't think we need the `attr_accessor` for `uncountable`. Let's define only `uncountable?`

Reply:
Sounds good. I fixed.
I need squash my commits?

Reply:
Yes please

Reply:
Squashed and rebased